cmd/migration/main.go

```diff
@@ -164,6 +170,103 @@ func main() {
 			}
 		}
 	}
+
+	extCredsSaved := []extCredsMigration{}
+	for _, si := range siSaved {
+		labels := map[string]string{"apbAction": "provision", "apbName": si.Spec.FQName}
+		extcredsJSON, err := etcdDao.GetRaw(fmt.Sprintf("/extracted_credentials/%v", si.ID))
+		if etcdDao.IsNotFoundError(err) {
+			continue
+		}
+		if err != nil {
+			revertExtractedCredentials(extCredsSaved)
+			revertJobState(jsSaved, options.MigrationNamespace)
+			revertBindInstance(biSaved)
+			revertServiceInstance(siSaved)
+			revertCrdSavedSpecs(crdSavedSpecs)
+			panic(fmt.Sprintf("Unable to migrate all extracted credentials - %v", err))
+		}
+
+		ec := apb.ExtractedCredentials{}
+		err = json.Unmarshal([]byte(extcredsJSON), &ec)
+		if err != nil {
+			revertExtractedCredentials(extCredsSaved)
+			revertJobState(jsSaved, options.MigrationNamespace)
+			revertBindInstance(biSaved)
+			revertServiceInstance(siSaved)
+			revertCrdSavedSpecs(crdSavedSpecs)
+			panic(fmt.Sprintf("Unable to migrate all extracted credentials - %v", err))
+		}
+		k8scli, err := clients.Kubernetes()
+		if err != nil {
+			revertExtractedCredentials(extCredsSaved)
+			revertJobState(jsSaved, options.MigrationNamespace)
+			revertBindInstance(biSaved)
+			revertServiceInstance(siSaved)
+			revertCrdSavedSpecs(crdSavedSpecs)
+			panic(fmt.Sprintf("Unable to migrate all extracted credentials - %v", err))
+		}
+		err = k8scli.SaveExtractedCredentialSecret(si.ID.String(), options.MigrationNamespace, ec.Credentials, labels)
+		if err != nil {
+			revertExtractedCredentials(extCredsSaved)
+			revertJobState(jsSaved, options.MigrationNamespace)
+			revertBindInstance(biSaved)
+			revertServiceInstance(siSaved)
+			revertCrdSavedSpecs(crdSavedSpecs)
+			panic(fmt.Sprintf("Unable to migrate all the binding instances - %v", err))
+		}
+		extCredsSaved = append(extCredsSaved, extCredsMigration{
+			ExtCreds:  ec,
+			ID:        si.ID.String(),
+			Namespace: options.MigrationNamespace,
+		})
+	}
+	for _, bi := range biSaved {
```

Comment:
We need a comment between the different sections:
```golang
//
// Convert Binding Instances
//
...
//
// Converting Extracted Credentials
//
...
```

Reply:
what's with the 3 commented lines?